lib/controller.js

```diff
@@ -747,18 +747,28 @@ controller.update = function(opts) {
 };
 
 controller.updateTesselWithVersion = function(force, tessel, currentVersion, build) {
+  function logCompletion() {
+    if (!force) {
+      logs.info('Updated', tessel.name, 'from ', currentVersion, ' to ', build.version);
+    } else {
+      logs.info('Force updated', tessel.name, 'to version', build.version);
+    }
+  }
 
```

Comment:
All this needs to work correctly is a line here that captures the present value of `tessel.name`, to be used further down in your call to `tessel.setName(...)`